exchanges/mock/server_test.go

```diff
@@ -38,89 +38,57 @@ func TestNewVCRServer(t *testing.T) {
 		Amount:   1,
 		Currency: "bitcoin",
 	})
-	if err != nil {
-		t.Fatal("marshal error", err)
-	}
+	require.NoError(t, err, "Marshal must not error")
 
 	testValue := HTTPResponse{Data: rp, QueryString: queryString, BodyParams: queryString}
 	test1.Routes["/test"][http.MethodGet] = []HTTPResponse{testValue}
 
 	payload, err := json.Marshal(test1)
-	if err != nil {
-		t.Fatal("marshal error", err)
-	}
+	require.NoError(t, err, "Marshal must not error")
 
 	err = os.WriteFile(testFile, payload, os.ModePerm)
-	if err != nil {
-		t.Fatal("marshal error", err)
-	}
+	require.NoError(t, err, "WriteFile must not error")
 
 	deets, client, err := NewVCRServer(testFile)
-	if err != nil {
-		t.Error("NewVCRServer error", err)
-	}
+	assert.NoError(t, err, "NewVCRServer should not error")
 
 	err = common.SetHTTPClient(client) // Set common package global HTTP Client
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 
 	_, err = common.SendHTTPRequest(t.Context(),
 		http.MethodGet,
 		"http://localhost:300/somethingElse?"+queryString,
 		nil,
 		bytes.NewBufferString(""), true)
-	if err == nil {
-		t.Error("Sending http request expected an error")
-	}
+	assert.Error(t, err, "SendHTTPRequest should return the correct error")
```

Comment:
🚧 I think this line was meant to be removed. The following line does the same thing, but better.
Also, the test message is inaccurate for the test.
```suggestion
```

Reply:
I don't know which version you are referring to.
The last commit had this line removed. Could you check it again?

Reply:
It does not look like that to me:
https://github.com/thrasher-corp/gocryptotrader/pull/1968/commits/def3f9aa6

And also here:
<img width="1227" height="799" alt="Screenshot 2025-08-25 at 19 57 53" src="https://github.com/user-attachments/assets/5cd4c3ec-5042-40f2-985d-8eb90f9b9fe9" />
78eb05118ecd17282bf3bb5723531fe